src/settings/SettingsStore.ts

```diff
@@ -516,6 +524,23 @@ export default class SettingsStore {
         return LEVEL_HANDLERS[level].isSupported();
     }
 
+    /**
+     * Determines if a setting supports a particular level.
+     * @param settingName The setting name.
+     * @param level The level.
+     * @returns True if supported, false otherwise. Note that this will not check to see if
```

Comment:
I think the types should also be specified here in curly brackets?

Reply:
That's only really applicable if we don't have TypeScript to tell us the types :)

Reply:
(it's easier to not delete the types when converting something from JS->TS, which is why the other functions have the types still)

Reply:
Ah, ok, I've actually never really thought about this much - thank you for the explanation!